### PR TITLE
Consulting - add Software audit

### LINKED ISSUE #300: Consulting Page - Software Auditing
Convert it into mdx file

image
**Figure:  Showing SSW Website Consulting page - Software Audit**

### PR DESCRIPTION
**Pull request by amankumarrr**
Table rendering is being implemented in https://github.com/SSWConsulting/SSW.Website-v3/issues/329#issue-1605920227

closes #300 

URL: `/consulting/software-audit`

### CODE REVIEW

File: components/util/consulting/benefits.tsx [hunk 1]
```diff
@@ -9,12 +9,14 @@ const BenefitCard = ({ data, aosType }) => {
...
       <figure className="relative float-left h-40 w-40 select-none">
-        <Image
-          src={`/images/benefits/${path.basename(image)}`}
-          sizes="100vw"
-          fill
-          alt="benefit icon"
-        />
+        {image && (
+          <Image
+            src={`/images/benefits/${path.basename(image)}`}
```
Comment (wicksipedia) [inline components/util/consulting/benefits.tsx]: Can we avoid hard coding the path and just use the one provided?

Comment (amankumarrr) [reply components/util/consulting/benefits.tsx]: Yes, it is possible to do that. Would you like me to create a variable to store the complete path of the image and then pass it as a prop to the Image component?

Comment (wicksipedia) [reply components/util/consulting/benefits.tsx]:
```suggestion
            src={image}
```
I would've expected this to work

Comment (amankumarrr) [reply components/util/consulting/benefits.tsx]: Fixed

Comment (wicksipedia) [reply components/util/consulting/benefits.tsx]: Sorry but you change still has a hardcoded path, it has only been moved around. Did you try the suggestion I posted?

Comment (wicksipedia) [reply components/util/consulting/benefits.tsx]: It looks like I already made this change on `main` (there are branch conflicts)

Comment (wicksipedia) [reply components/util/consulting/benefits.tsx]: I've merged changes from `main` as the method works and avoids hardcoded paths

Comment (amankumarrr) [reply components/util/consulting/benefits.tsx]: I read it after pushing my change

> It looks like I already made this change on `main` (there are branch conflicts)

**Comment by wicksipedia** [comment]
couple of bugs to sort out:
image

**Comment by amankumarrr** [comment]
> couple of bugs to sort out: image

Broken image has been fixed

**Review by wicksipedia — APPROVED** [review]
lgtm - should be good once the table issue is done

**Pull request merged**